db/schema.rb

```diff
@@ -13,7 +13,7 @@
 
 ActiveRecord::Schema.define(version: 20141005040238) do
 
-  create_table "widgets", force: true do |t|
+  create_table "widgets", force: :cascade do |t|
```

Comment:
huh, I haven't seen this before. Do you know what this does or was it automatic?